spec/features/notes_on_merge_requests_spec.rb

```diff
@@ -17,8 +17,8 @@
 
     describe "the note form" do
       it 'should be valid' do
-        should have_css(".js-main-target-form", visible: true, count: 1)
-        find(".js-main-target-form input[type=submit]").value.should == "Add Comment"
+        is_expected.to have_css(".js-main-target-form", visible: true, count: 1)
+        expect(find(".js-main-target-form input[type=submit]").value).to eq("Add Comment")
```

Comment:
Line is too long. [90/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.